pywinauto/controls/common_controls.py

```diff
@@ -1028,23 +1028,23 @@ def rectangle(self, text_area_rect = True):
         del remote_mem
         return rect
     # Non PEP-8 alias
-    Rectangle = rectangle
+    ClientRect = client_rect

```

Comment:
```HwndWrapper.client_rect``` is aliased by ```ClientRect``` so I prefer to leave this alias too. In fact, ```_treeviewelement``` is an internal class that I think shouldn't be exposed to the user. It even doesn't inherit from ```BaseWrapper```. So there is a more work to be done here later